ansible/roles/rke/vars/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
-kubectl_version: v1.15.2
-helm_version: v2.16.1
-rancher_cli_version: v2.2.0
+kubectl_version: v1.17.12
+helm_version: v3.2.1
```

Comment:
Any reason not to use the latest version here? (v1.19.0 and v3.3.4, respectively)

Reply:
not a good one, fixed